yoti-sdk-sandbox/src/test/java/com.yoti.api.client.sandbox/profile/request/TestData.java

```diff
@@ -0,0 +1,86 @@
+package com.yoti.api.client.sandbox.profile.request;
+
+import com.yoti.api.client.HumanProfile;
+import com.yoti.api.client.sandbox.profile.request.attribute.Attribute;
+import com.yoti.api.client.sandbox.profile.request.attribute.ProfileAttributeName;
+import com.yoti.api.client.sandbox.profile.request.attribute.SandboxDateAttribute;
+import com.yoti.api.client.sandbox.profile.request.attribute.derivation.AgeDerivationSupported;
+import com.yoti.api.client.sandbox.profile.request.attribute.derivation.AgeVerification;
+
+import org.bouncycastle.util.encoders.Base64;
+
+class TestData {
+
+    static final String FAMILY_NAME = "Doe";
+    static final String GIVEN_NAMES = "John lol";
+    static final String EMAIL_ADDRESS = "john@email.com";
+    static final String FULL_NAME = "John lol Doe";
+    static final String DATE_OF_BIRTH = "1902-02-02";
+    static final String NATIONALITY = "InTheMiddleOfNowhere";
+    static final String PHONE_NUMBER = "123456789";
+    static final String POSTAL_ADDRESS = "Road of the roads";
+    static final String SELFIE = "12345678987654321234567876543212345676543211234";
+
```

Comment:
Any support for structured postal address?

Reply:
Yep, I'll add a test for that also, sorry about that.